src/main/java/org/jabref/gui/preferences/customentrytypes/CustomEntryTypesTab.java

```diff
@@ -176,7 +177,25 @@ private void setupFieldsTable() {
         makeRotatedColumnHeader(fieldTypeColumn, Localization.lang("Required"));
 
         fieldTypeMultilineColumn.setCellFactory(CheckBoxTableCell.forTableColumn(fieldTypeMultilineColumn));
-        fieldTypeMultilineColumn.setCellValueFactory(item -> item.getValue().multilineProperty());
+        fieldTypeMultilineColumn.setCellValueFactory(item -> {
+            BooleanProperty property = item.getValue().multilineProperty();
+            property.addListener((obs, wasSelected, isSelected) -> {
+                if (isSelected) {
+                    viewModel.entryTypes().forEach(typeViewModel -> {
+                        typeViewModel.fields().stream()
+                                     .filter(f -> f.displayNameProperty().get().equals(item.getValue().displayNameProperty().get()))
+                                     .forEach(f -> f.multilineProperty().set(true));
+                    });
+                } else {
+                    viewModel.entryTypes().forEach(typeViewModel -> {
+                        typeViewModel.fields().stream()
+                                     .filter(f -> f.displayNameProperty().get().equals(item.getValue().displayNameProperty().get()))
+                                     .forEach(f -> f.multilineProperty().set(false));
```

Comment:
The variable `isSelected` can be used instread of `false` (and `true` above). - All other code is same.

Reply:
I am not too sure why I went this route but 1ca84f7b63fa2a2d1652a7dd35ed3fec82c172cd fixes this issue and removes duplicate code as suggested. Thanks!